### DISCUSSION TITLE
Encryption

### DISCUSSION THREAD

**Discussion by klaids** [jgromes/RadioLib#383] [Q&A]
Goodmorning, 
Is it possibile to add a Encryption funtions directly on RadioLib? If i would to use Encryption from my data before send a LoRa packet, Which/There is a library to implement this on STM32?
Best Regards

**Comment by jgromes** [comment]
I don't think it's a good idea to do this directly within RadioLib. Basically, I don't see a reason why this should be included, when libraries that implement this feature exists and can be run on any platform that is supported by RadioLib (e.g. https://github.com/kokke/tiny-AES-c).

**Reply by klaids** [reply]
Ok, thanks a lot for your answer.
